tests/unit/test_units.py

```diff
@@ -0,0 +1,120 @@
+import numpy as np
+import pytest
+
+from openscm.units import unit_registry
+from openscm.units import DimensionalityError, UndefinedUnitError, UnitConverter
+
+
+def test_unit_registry():
+    CO2 = unit_registry("CO2")
+    assert CO2.to("C").magnitude == 12.0 / 44.0
```

Comment:
```suggestion
    assert CO2.to("C").magnitude == 12 / 44
```

Reply:
Use same accuracy as below?

Also, should this be some kind of almost_equal?

Reply:
@swillner explanation here is that Python3 does floating point division by default

Reply:
yeah, I used explicit `round` somewhere where I thougt it's necessery... There is also an `assertAlmostEqual` in `unittest` standard package. Should we agree on a standard here?